libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -269,10 +269,21 @@ class AC_PosControl
     ///     throttle targets will be sent directly to the motors
     void update_vel_controller_xyz();
 
-    /// get desired roll, pitch which should be fed into stabilize controllers
+    /// get desired roll and pitch to be passed to the attitude controller
     float get_roll() const { return _roll_target; }
     float get_pitch() const { return _pitch_target; }
 
+    /// get desired yaw and yaw rate to be passed to the attitude controller
+    float get_yaw_cd() const { return _yaw_target; }
+    float get_yaw_rate_cds() const { return _yaw_rate_target; }
+
+
+    /// get desired roll and pitch to be passed to the attitude controller
+    Vector3f get_thrust_vector() const;
+
+    // lean_angles_to_accel - convert roll, pitch lean angles to lat/lon frame accelerations in cm/s/s
+    bool calculate_yaw_rate_yaw();
```

Comment:
do we want to have "yaw" twice in this name?

Reply:
What should we change it to?
calculate_yaw_angle_and_rate?